e2e/txpool_test.go

```diff
@@ -251,19 +226,40 @@ func TestTxPool_TransactionCoalescing(t *testing.T) {
 		return msg
 	}
 
+	// testTransaction is a helper structure for
+	// keeping track of test transaction execution
+	type testTransaction struct {
+		txHash web3.Hash // the transaction hash
+		block  *uint64   // the block the transaction was included in
+	}
+
+	testTransactions := make([]*testTransaction, 0)
+
 	// Add the transactions with the following nonce order
 	nonces := []uint64{0, 2}
 	for i := 0; i < len(nonces); i++ {
```

Comment:
How about creating new function that generate&send tx and append to `textTransactions`?

Reply:
We dropped this test completely in PR #526, so I don't see any use for adding it